packaging/centos7/cloud.spec

```diff
@@ -418,6 +424,8 @@ fi
 # Install mysql-connector-python
 pip3 install %{_datadir}/%{name}-management/setup/wheel/six-1.15.0-py2.py3-none-any.whl %{_datadir}/%{name}-management/setup/wheel/setuptools-47.3.1-py3-none-any.whl %{_datadir}/%{name}-management/setup/wheel/protobuf-3.12.2-cp36-cp36m-manylinux1_x86_64.whl %{_datadir}/%{name}-management/setup/wheel/mysql_connector_python-8.0.20-cp36-cp36m-manylinux1_x86_64.whl
 
+pip3 install urllib3
```

Comment:
@DaanHoogland why do we need it for cloudstack-management pkg?

Reply:
I'll double check, I think it's an indirect dependency, Also it is not packaged which is mayby not what we want , I would mean internet has to be available during install. I'll look into this.